### PR TITLE
MOB-537: Add CEX transfer warning.

### PR DESCRIPTION
**Pull request by prashanDYDX**
Screenshot 2024-06-05 at 7 32 00 PM
Screenshot 2024-06-05 at 7 31 47 PM
Screenshot 2024-06-05 at 7 31 36 PM

### CODE REVIEW

File: v4/feature/receipt/src/main/java/exchange/dydx/trading/feature/receipt/DydxReceiptView.kt [hunk 1]
```diff
@@ -180,6 +184,10 @@ object DydxReceiptView : DydxComponent {
                         }
                     }
                 }
+
+                item {
+                    Spacer(modifier = Modifier.height(ThemeShapes.VerticalPadding))
```
Comment (prashanDYDX) [inline v4/feature/receipt/src/main/java/exchange/dydx/trading/feature/receipt/DydxReceiptView.kt:189]: Bottom padding from the modifier not working for some reason...

File: v4/feature/transfer/src/main/java/exchange/dydx/trading/feature/transfer/transferout/DydxTransferOutViewModel.kt [hunk 2]
```diff
@@ -173,6 +174,17 @@ class DydxTransferOutViewModel @Inject constructor(
                     }
                 },
             ),
+            transferMemo = TransferMemoBox.ViewState(
+                localizer = localizer,
+                value = transferInput?.memo,
+                showCexWarning = transferInput?.token != USDC_TOKEN && (transferInput?.memo?.isEmpty() ?: true),
```
Comment (prashanDYDX) [inline v4/feature/transfer/src/main/java/exchange/dydx/trading/feature/transfer/transferout/DydxTransferOutViewModel.kt]: in hindsight this should've been done in Abacus also

**Comment by ruixhuang** [comment]
Mike has made the change so that Destination and Network are side-by-side.   Can we do that too ?https://github.com/dydxprotocol/v4-native-ios/pull/184

File: v4/platformUI/src/main/java/exchange/dydx/platformui/components/alerts/PlatformInlineAlert.kt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+) {
+    Row(
+        modifier
+            .clip(RoundedCornerShape(4.dp))
+            .background(level.backgroundColor)
+            .height(IntrinsicSize.Min),
+    ) {
+        Box(
+            Modifier
+                .width(3.dp)
```
Comment (ruixhuang) [inline v4/platformUI/src/main/java/exchange/dydx/platformui/components/alerts/PlatformInlineAlert.kt]: I think it's 6.dp here.  https://www.figma.com/design/mKevZOfE9nj6MZpiolKYW1/dYdX-%E2%80%BA-Mobile?m=dev&node-id=5956-15063&t=HOX2R4ZpI1AKKQX6-1

**Comment by prashanDYDX** [comment]
Screenshot 2024-06-06 at 6 52 16 PM

@ruixhuang new layout screenshot

**Review by ruixhuang — APPROVED** [review]

**Pull request merged**